test/e2e/mcq.cy.js

```diff
@@ -1,26 +1,26 @@
 describe('Multiple Choice Question', function () {
   beforeEach(function () {
-    cy.getData()
+    cy.getData();
     cy.visit('/');
   });
 
   it('should display the mcq component', function () {
-    const mcqComponents = this.data.components.filter((component) => component._component === 'mcq')
-    mcqComponents.forEach((mcqComponent) => {
+    const mcqComponents = this.data.components.filter(component => component._component === 'mcq');
+    mcqComponents.forEach(mcqComponent => {
       cy.visit(`/#/preview/${mcqComponent._id}`);
-      const bodyWithoutHtml = cy.helpers.stripHtml(mcqComponent.body)
-      cy.testContainsOrNotExists('.mcq__body', bodyWithoutHtml)
+      const stripHtml = cy.helpers.stripHtml;
+      cy.testContainsOrNotExists('.mcq__body', stripHtml(mcqComponent.body));
+      cy.testContainsOrNotExists('.mcq__title', stripHtml(mcqComponent.displayTitle));
+      cy.testContainsOrNotExists('.mcq__instruction', stripHtml(mcqComponent.instruction));
+
+      cy.get('.mcq-item').should('have.length', mcqComponent._items.length);
+      mcqComponent._items.forEach(item => {
+        cy.testContainsOrNotExists('.mcq-item__text', item.text);
```

Comment:
```suggestion
        cy.testContainsOrNotExists('.mcq-item__text', stripHtml(item.text));
```

Also compiled html 
https://github.com/adaptlearning/adapt-contrib-mcq/blob/f585154e6778e065855bea97365c1792f0b8af2e/templates/mcq.jsx#L109